engines/director/tests.cpp

```diff
@@ -132,7 +132,7 @@ void Window::testFontScaling() {
 	Common::Event event;
 
 	while (true) {
-		if (g_system->getEventManager()->pollEvent(event))
+		if (g_director->pollEvent(event))
```

Comment:
I believe this could be reverted :D

Reply:
Not sure I follow? The purpose behind wrapping the eventmanager with DirectorEngine::pollEvent was to make it possible for UnitTest to inject fake events